grails-app/conf/application.yml

```diff
@@ -180,8 +183,8 @@ security:
 
     bypass: false # set to true for non-ALA deployment
     gateway: false
-    uriExclusionFilterPattern: '/assets.*,/images.*,/css.*,/js.*,/less.*,/tasks/status/.*'
-    uriFilterPattern: '/alaAdmin.*,/log/,/log'
+    uriExclusionFilterPattern: '/assets.*,/images.*,/css.*,/js.*,/less.*,'
+    uriFilterPattern: '/alaAdmin.*'
```

Comment:
Has the `/log` endpoint been removed? If not then it will now be public and we normally don't want logs to be public...